package.json

```diff
@@ -81,8 +81,11 @@
     "@rollup/rollup-linux-x64-gnu": "^4.17.2"
   },
   "lint-staged": {
-    "*.js": "npm run lint:js",
-    "*.css": "npm run lint:css",
-    "**/*": "npm run lint:ls && npm run format"
+    "*.js": [
```

Comment:
Interesting, didn't know about the array syntax, that makes sense.  I get the `--` part but curious how it works for say this case?
```sh
# before
"*.css": "npm run lint:css",

# after
"*.css": "npm run lint:css --",
```

How come we would need the `--` here?  Does. this imply `format` is being run implicitly and so needs to be chained in each entry in the lint staged config?

Reply:
Honestly, I'm not totally clear with why the `--` is needed. It was listed in the examples so I assume it's something within `lint-staged` that requires it.

Reply:
With `npm` you use the `--` to forward arguments to the NPM CLI, like here
https://github.com/thescientist13/gallinago/blob/master/package.json#L30
```
"test": "c8 node --debug-port 3333 ./node_modules/mocha/bin/mocha",
"test:tdd": "npm test -- --watch"
```

I'm wondering if this just enabling some sort of catch-all forwarding here?